apis/server/image_bridge.go

```diff
@@ -75,9 +76,12 @@ func (s *Server) getImage(ctx context.Context, rw http.ResponseWriter, req *http
 }
 
 func (s *Server) listImages(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
-	filters := req.FormValue("filters")
+	filter, err := filters.FromParam(req.FormValue("filters"))
```

Comment:
any reason to change `filters`  to `filter` ?

Reply:
how about this? change filters into apifilters? just ask.